test/src/hitbox_test.cpp

```diff
@@ -23,3 +24,24 @@ TEST (HitboxTest, simple_hitbox_overlapping_false) {
     bool actual = hitbox1.is_overlapping(hitbox2);
     EXPECT_EQ(expected, actual);
 }
+
+TEST (HitboxTest, constructor_1) {
+    Hitbox hitbox1 = Hitbox(0, 0, 100, 100);
+    bool actual, expected = true;
+    if (hitbox1.hitbox().x == 0 && hitbox1.hitbox().y == 0 && hitbox1.hitbox().w == 100 && hitbox1.hitbox().h == 100)
+    	actual = true;
+    else 
+    	actual = false;
+    EXPECT_EQ(expected, actual);
+}
+
+TEST (HitboxTest, constructor_2) {
+    SDL_Rect rect1 = {0, 0, 100, 100};
+    Hitbox hitbox1 = Hitbox(rect1);
+    bool actual, expected = true;
+    if (hitbox1.hitbox().x == 0 && hitbox1.hitbox().y == 0 && hitbox1.hitbox().w == 100 && hitbox1.hitbox().h == 100)
+    	actual = true;
+    else
+    	actual = false;
+    EXPECT_EQ(expected, actual);

```

Comment:
It would probably better to have a `EXPECT_EQ` for every member. Also, you can probably keep this one more sparse without the `actual` and `expected`. I know i wrote that before, but it's not really necessary. 

So I would imagine it would be more like:
```c++
EXPECT_EQ(hitbox1.hitbox().x, 0);
EXPECT_EQ(hitbox1.hitbox().y, 0);
...
```